drivers/i3c/Kconfig.stm32

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 EXALT Technologies.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module = I3C_STM32
+module-str = i3c_stm32
+
+source "subsys/logging/Kconfig.template.log_config"
+config I3C_STM32
+	bool "STM32 I3C driver support"
+	depends on DT_HAS_ST_STM32_I3C_ENABLED
+	select USE_STM32_HAL_I3C
```

Comment:
I confirm this is not required. Please remove

Reply:
@erwango  We chose to use LL instead of HAL, as we took i2c_stm32 as a reference and assumed that following the LL convention would align better with the project's standards.